specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -523,16 +528,16 @@
    "metadata": {},
    "source": [
     "### 4b) Compute HDD and CDD \n",
-    "We'll use the climakitae helper function `compute_hdd_cdd` to perform the computation, which uses a default standard temperature of 65 degrees F. You can change this default using the function argument `standard_temp`. The function performs the following calculations:<br><br>\n",
-    "**HDD = 65 - temperature<br>\n",
-    "CDD = (-1)\\*(65 - temperature)**<br><br>\n",
-    "For HDD, we can just subtract the 2m temperature from 65 degrees Farenheight, then set any negative to 0. For CDD, we will do the same, but will then multiply by -1 to turn negative values to positive, then set negative values to 0. We need to multiply by -1 for CDD to avoid having all negative values; for example, on a day of 80F, CDD = 65 - 80 = -15, but the CDD value is +15. Multiplying -15 by -1 will give us the true value of 15. "
+    "We'll use the climakitae helper function `compute_hdd_cdd` to compute both heating and cooling degree days, which uses a default temperature of 65 degrees F for both metrics. You can modify the default options using the function arguments `hdd_threshold` and `cdd_threshold` to represent any threshold of your choosing. In the example below, we will calculate HDD with a threshold of 60 degF and CDD with a threshold of 70 degF. The function performs the following calculations:<br><br>\n",
```

Comment:
The hdd/cdd and hdh/cdh functions now accept thresholds for different thresholds, but I've also removed any reference to 65 degF, except that it is the default value. Do we want to modify the function so that the user must pass a value instead and remove all 65 degF references?